packages/core/src/stepper/stepper-prev-trigger.tsx

```diff
@@ -0,0 +1,43 @@
+import { composeEventHandlers } from "@kobalte/utils";
+import { type Component, type ValidComponent, splitProps, JSX, Show } from "solid-js";
+import * as Button from "../button";
+import { type ElementOf, type PolymorphicProps } from "../polymorphic";
+import { useStepperContext } from "./stepper-context";
+
+export interface StepperPrevTriggerOptions { }
+
+export interface StepperPrevTriggerCommonProps<T extends HTMLElement = HTMLElement> {
+	onClick?: JSX.EventHandlerUnion<T, MouseEvent>;
+}
+
+export interface StepperPrevTriggerRenderProps
+	extends StepperPrevTriggerCommonProps,
+	Button.ButtonRootRenderProps { }
+
+export type StepperPrevTriggerProps<T extends ValidComponent | HTMLElement = HTMLElement> =
+	StepperPrevTriggerOptions & Partial<StepperPrevTriggerCommonProps<ElementOf<T>>>;
+
+export function StepperPrevTrigger<T extends ValidComponent = "button">(
+	props: PolymorphicProps<T, StepperPrevTriggerProps<T>>
+) {
+	const context = useStepperContext();
+	const [local, others] = splitProps(props as StepperPrevTriggerProps, ["onClick"]);
+
+	const onClick: JSX.EventHandlerUnion<HTMLButtonElement, MouseEvent> = (e) => {
+		if (context.step() > 0) {
+			context.setStep(context.step() + 1);
```

Comment:
```suggestion
			context.setStep(context.step() - 1);
```

Reply:
Ah, my bad! I think that was a typo—thanks for catching it!

Reply:
Yeah! That's why I suggested adding functions like `moveForward` and `moveBackward` to the context. You'll reduce the number of human typos like this!